src/components/ec-panel/ec-panel.vue

```diff
@@ -2,43 +2,69 @@
   <aside
     v-if="show"
     class="ec-panel"
-    data-test="ec-panel"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-panel` : 'ec-panel'"
   >
-    <div class="ec-panel__header">
-      <div class="ec-panel__header-icons">
-        <a
-          v-if="isBackEnabled"
-          aria-label="Go back"
-          class="ec-panel__header-action ec-panel__header-action--back"
-          href="#"
-          data-test="ec-panel__header-action--back"
-          @click.stop.prevent="goBack"
+    <div
+      class="ec-panel__content"
+      data-test="ec-panel__content"
+    >
+      <div
+        class="ec-panel__header"
+        data-test="ec-panel__header"
+      >
+        <div
+          class="ec-panel__header-icons"
+          data-test="ec-panel__header-icons"
         >
-          <ec-icon
-            class="ec-panel__header-icon"
-            name="simple-chevron-left"
-            :size="24"
-          />
-        </a>
-        <a
-          aria-label="Close panel"
-          class="ec-panel__header-action ec-panel__header-action--close"
-          href="#"
-          data-test="ec-panel__header-action--close"
-          @click.stop.prevent="closePanel"
+          <a
+            v-if="isBackEnabled"
+            aria-label="Go back"
+            class="ec-panel__header-action ec-panel__header-action--back"
+            href="#"
```

Comment:
probably I've asked this before, if this isn't a link, shall we make it a button?

Reply:
That was done many months ago. Amended anyway